userguide/intro.rst

```diff
@@ -288,22 +288,19 @@ called *amd64* by AMD and *Intel 64* by Intel.
    firmware interface or EFI.
 
 Actual hardware requirements vary depending on the workflow of your
-%brand% system. This section provides some starter guidelines. The
-`FreeNAS® Hardware Forum
+%brand% system. This section provides some starter guidelines.
+`The Official FreeNAS® Hardware Guide
```

Comment:
Since this is for v12... should this be TrueNAS Hardware Guide?  I assume we're going to be renaming the older one

Reply:
I could rename it here, but the link will open a page that is still titled "FreeNAS Hardware guide" and I don't know if that post will be renamed...